packages/react-native-layout/src/components/container/Container.tsx

```diff
@@ -9,10 +9,12 @@ const Container: React.FC<ContainerProps & TestableComponent> = ({
   testID,
   type,
   backgroundColor = 'transparent',
+  flex,
```

Comment:
I would pass the style prop and spread it, that way this component can also be styled with styled components 
```const MyContainer = styled(Container)`` ```

Reply:
That would be nice, but let's add that to engineering chapter backlog?

Otherwise we have scope creep in this quick "bugfix" PR

Reply:
@JaccoGoris 

The point of this component is single-purpose. If you want to style it with styled components, it has failed in it's purpose. :-)

Reply:
all it is is exposing the style prop and spreading it haha, not a big deal in terms of scope creep I think. My thoughts, I have worked with so many component libraries and the power of them is that they do a lot of complex things already. then you want to add a little padding to something and you can't because the author of the library chose not to expose that part.

Inversion of control is an important pattern in the pursuit of keeping clean code flexible and easy to work with. Therefore, I'm always more a fan of letting the consumer decide the styling and not lock it in to what the component author thinks is best (you can't know in advance)

also literally every component can be styled out of the box, why take this away from your users and force them to wrap your component with a wrapper component in order to add a border or a spacing? bed for readability and not a good DX.

Reply:
Also, this PR wouldn't have been a thing if it already supported the standard style prop because you could have set flex in your styling, as normal. (very consistent)

Reply:
Could you list a specific use case you want to achieve with the **Padding** component styling?

The premise of the "simplicity" of these kind of **layout** components is that they are undeniably simple to use. It's also very opinionated. Adding inversion of control to these kinds of components only  allows for the kind of flexibility that breaks the component at its core or just do things the component was not intended for.

Reply:
The way I see these components is that they add a convenient api to not have to style it yourself. so they add some functionality on top of a normal View. which is great. You don't have to protect your users here I feel. Your attempting to add convenience here. limiting what a normal View can do in exchange for a couple of props that write very specific css for me is a very tight tradeoff for me. I just don't see the harm is exposing a style prop, which all other things expose, ever complex components like scrollviews and such

Reply:
I don;t see the value in going against the conventions of the ecosystem